dvc/command/ls.py

```diff
@@ -39,14 +39,12 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawTextHelpFormatter,
     )
     list_parser.add_argument(
-        "url",
-        help="Supported urls:\n"
-        "/path/to/file\n"
-        "/path/to/directory\n"
-        "C:\\\\path\\to\\file\n"
-        "C:\\\\path\\to\\directory\n"
-        "https://github.com/path/to/repo\n"
-        "git@github.com:path/to/repo.git\n",
+        "url", help="Location of DVC or Git repository to download from"
+    )
+    list_parser.add_argument(
+        "target",
+        nargs="?",
+        help="Path to directory within the repository to list outputs for",
```

Comment:
@JIoJIaJIu I moved this up bc it seems more important than the following args (options). It won't change the command behavior or help output right?

Reply:
Hello @jorgeorpinel , sorry for the late reply
Yes, it's ok to move up